docs/content/reference/operating_systems_support.md

```diff
@@ -7,9 +7,14 @@ weight: 10
 tags: ["setup", "reference", "windows", "linux"]
 ---
 
-The Infection Monkey project supports many popular OSes (but we are always interested in supporting more).
+The Infection Monkey project supports many popular operating systems.
 
-The Infection Monkey Agent has been tested to run on the following operating systems (on the x86_64 architecture):
```

Comment:
Why remove the architecture?

Reply:
The dev who built the ransomware wallpaper feature actually ran into trouble where he was using the wrong architecture. Specifying the architecture here is important.